src/app/store/effects/upload.effects.ts

```diff
@@ -132,6 +143,24 @@ export class UploadEffects {
       });
   }
 
+  uploadNewImage(file: File): void {
+    this.contentService.getNodeInfo().subscribe((node) => {
```

Comment:
can we use the await? and try catch for the error

Reply:
changed with better rxjs approach